src/Main.hx

```diff
@@ -48,20 +50,25 @@ class Main {
 	{
 		if (!updatePromises.exists(local)) {
 			updatePromises[local] = new Promise(function(resolve, reject) {
-				trace("updating: fetching");
-				fetch(local, function (ferr, stdout, stderr) {
+				trace('INFO: updating: fetching from $remote');
```

Comment:
I think this needs to be filtered a bit: `remote` will contain password/token info.

Note: when dealing with tokens, they can appear after the `:` (as a password would), [before the `:`](https://github.com/blog/1270-easier-builds-and-deployments-using-git-over-https-and-oauth) or both

Reply:
Oh, and this is awesome, thanks a lot!

Reply:
Hmm, yes, in general, this should be sanitized. In my special use-case, clone tokens are already sanitized by GitLab....  Could something like `params.repo` just work?